interface/src/Application.cpp

```diff
@@ -641,6 +646,9 @@ Application::Application(int& argc, char** argv, QElapsedTimer &startup_time) :
     auto applicationUpdater = DependencyManager::get<AutoUpdater>();
     connect(applicationUpdater.data(), &AutoUpdater::newVersionIsAvailable, dialogsManager.data(), &DialogsManager::showUpdateDialog);
     applicationUpdater->checkForUpdate();
+
+    auto& packetReceiver = nodeList->getPacketReceiver();
```

Comment:
AFAIK, unfortunately, it doesn't:
> in C++11, auto defaults to being by-value for references
http://www.cprogramming.com/c++11/c++11-auto-decltype-return-value-after-function.html

Also I tried this:
```
#include <iostream>

class Obj {
public:
    Obj() { }
    Obj(const Obj& other) {
        std::cout << "Obj copy ctor called" << std::endl;
    }
};

int main() {
    std::cout << "Creating o1" << std::endl;
    Obj o1;
    std::cout << "Creating o2" << std::endl;
    auto& o2 = o1;
    std::cout << "Creating o3" << std::endl;
    auto o3 = o1;
}
```
```
» ./a.out
Creating o1
Creating o2
Creating o3
Obj copy ctor called
```